index.html

```diff
@@ -1,10 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="author" content="Kaboha Jean Mark">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Basic HTML Boilerplate</title>
-        <link rel="stylesheet" href="css/styles.css">
-    </head>
-</html>
\ No newline at end of file
+<head>
+    <meta charset="UTF-8" />
+    <meta name="author" content="Kaboha Jean Mark">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pele Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body id="main">
+  <header>
+    <div class="content-wrap">
```

Comment:
```suggestion
    <section class="content-wrap">
```
Perhaps it would be nice to have all of this content in a section tag for more semantic HTML5 and improving the SEO